setup.py

```diff
@@ -194,7 +194,7 @@ def extract_version():
 
 
 setup(
-    name='Iris',
+    name='iris-scitools',
```

Comment:
@lbdreyer `pypi` uses a **weighted** sort for packages on search, then defaults to alphabetical order for equal weighted packages i.e. the `occurrence of search term weighted by field (name, summary, keywords, description, author, maintainer)` e.g see [here](https://pypi.python.org/pypi?%3Aaction=search&term=iris&submit=search)... so I guess *whatever* name we use, we should liberally sprinkle that name throughout the metadata kwargs passed to `distutils.core.setup` in order to raise our `iris` to the top of the `pypi` search query results based on it's **weight** ...

btw I went for `scitools-iris` because that's the order in `https://github.com/SciTools/iris` and `http://scitools.org.uk/iris/index.html` ...

Do any other @SciTools/iris-devs have any thoughts/opinions? So we can go with the wisdom of the crowd ...